talk/owt/BUILD.gn

```diff
@@ -8,21 +8,20 @@ import("//testing/test.gni")
 # Introduced for using libvpx config files. We only enable libvpx rate
 # controller for VP9 on Windows.
 if (is_win) {
-  os_category = current_os
-
-  if (current_cpu == "x86") {
-    cpu_arch_full = "ia32"
-  } else if (current_cpu == "x64") {
-    cpu_arch_full = "x64"
-  } else if (current_cpu == "arm") {
-    if (arm_use_neon) {
-      cpu_arch_full = "arm-neon"
-    } else if (is_android) {
-      cpu_arch_full = "arm-neon-cpu-detect"
-    } else {
-      cpu_arch_full = "arm"
-    }
-  }
+ 
+ if (current_cpu == "x86") {
+   cpu_arch_full = "ia32"
+ } else if (current_cpu == "x64") {
+   cpu_arch_full = "x64"
+ } else if (current_cpu == "arm") {
+   if (arm_use_neon) {
+     cpu_arch_full = "arm-neon"
+   } else if (is_android) {
```

Comment:
under is_win, there is no possibility for is_android to be defined. Although this is not your issue, can you remove it?